### PR TITLE
fix tfa error

### PR DESCRIPTION
**Pull request by tristanhcole**
fix tfa error caused by typo

### CODE REVIEW

File: app/client/sagas/authSagas.js [hunk 1]
```diff
@@ -464,8 +464,7 @@ function* watchInviteUserRequest() {
 function* validateTFA({ payload }) {
   try {
     const validateTFAresponse = yield call(ValidateTFAAPI, payload);
-
-    yield call(updateOrganisationStateFromLoginData, activated_account);
```
Comment (tristanhcole) [inline app/client/sagas/authSagas.js]: `activated_account ` was undefined, causing the saga to throw an error. We should implement basic frontend tests for all core paths

**Comment by tristanhcole** [comment]
Why is the version wrong?

**Comment by enjeyw** [comment]
> Why is the version wrong?

master is 1.1.4 no?

**Comment by tristanhcole** [comment]
Frontend and backend are on different versioning
https://github.com/teamsempo/SempoBlockchain/blob/master/app/package.json

**Comment by enjeyw** [comment]
> Frontend and backend are on different versioning
> https://github.com/teamsempo/SempoBlockchain/blob/master/app/package.json

ohhh yepyep forgot!

**Pull request merged**